ubuntu1704-desktop.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_comment": "Build with `packer build -var-file=ubuntu1704-desktop.json ubuntu.json`",
+  "vm_name": "ubuntu1704-desktop",
+  "desktop": "true",
+  "locale": "en_US.UTF-8",
+  "cpus": "1",
+  "disk_size": "130048",
```

Comment:
Values are the same like in the other ubuntu json files. So if they are shady here they are shady in all the files too. Issue should be discussed with the people responsible for the values and should be fixed in all the other files too.